index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Aoamos">
+    <meta name="description" content="A project after watching 4hours class">
+    <title>Little Taco Shop</title>
+    <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="style.css" type="text/css">
+</head>
+
+<body>
+    <header>
+        <h2>Welcome to The Little Taco Shop</h2>
+        <nav>
+            <ul>
+                <li><a href="#about">About <abbr title="Little Taco Shop">LTS</abbr></a></li>
+                <li><a href="#menu">Our Menu </a></li>
+                <li><a href="other/hours.html">Store Hours</a></li>
+                <li><a href="other/contact.html">Contact Us </a></li>
+            </ul>
+        </nav>
+        <figure>
```

Comment:
Always remove your ordered list icons from your page and makes sure to center all your elements
look at your work
![bullet](https://github.com/Aoamos/Complete_html/assets/108594742/95b49391-9cb3-432a-be52-fb4938b45884)

try to center all the elements. though it is good to follow tutorials but learn to add your own ideas at all times that's what makes you a developer. except if such project has a list of requirements.

do this and resubmit for review. 

happy coding